recipe_scrapers/mob.py

```diff
@@ -72,10 +72,12 @@ def instructions(self):
         )
 
     def cuisine(self):
-        return self.recipe_json["cuisines"][0]["title"]
+        return " & ".join(
```

Comment:
For consistency with a few other scrapers, let's use a comma-separated list for multiple categories here.  Anecdotally, that seems to be the way that many recipe sites list multi-categories too.

```suggestion
        return ", ".join(
```

Reply:
@jayaddison Sure, done!

Reply:
Thanks again!

Reply:
Ah, it looks like I commented on the wrong line here: this was the `cuisine` field, but I think we want these comma-separated lists for both `cuisine` and `category`.

(sorry, I should have noticed that during the earlier review)

Reply:
no problem, updated now 👍